plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt

```diff
@@ -325,17 +412,22 @@ class FeatureDevController(
                 }
             }
 
+            val rejectedFilesCount = filePaths.count { it.rejected } + deletedFiles.count { it.rejected }
+            val acceptedFilesCount = filePaths.count { it.changeApplied } + filePaths.count { it.changeApplied }
+            val remainingFilesCount = filePaths.count() + deletedFiles.count() - acceptedFilesCount - rejectedFilesCount
+
             AmazonqTelemetry.isAcceptedCodeChanges(
-                amazonqNumberOfFilesAccepted = (filePaths.filterNot { it.rejected }.size + deletedFiles.filterNot { it.rejected }.size) * 1.0,
+                amazonqNumberOfFilesAccepted = remainingFilesCount * 1.0,
```

Comment:
why are we multiplying this by 1.0?

Reply:
Coercing to double; updating to `.toDouble()` to be more explicit.